toolkit/tools/imagecustomizer/README.md

```diff
@@ -0,0 +1,71 @@
+# Mariner Image Customizer
+
+The Mariner Image Customizer (MIC) is a tool that can take an existing generic Mariner
+image and modify it to be suited for particular scenario.
+
+MIC uses [chroot](https://en.wikipedia.org/wiki/Chroot) (and loopback block devices) to
+customize the image.
+This is the same technology used to build the Mariner images (along with most other
+Linux distros).
+This is in contrast to some other image customization tools, like Packer, which
+customize the image by booting it inside a VM.
+
+There are a number of advantages and disadvantages to the `chroot` approach to
+customizing images.
+
+Advantages:
+
+- Lower overhead, since you don't need to boot up and shutdown the OS.
+- More precision when making changes, since you won't see any side effects that come
+  from the OS running.
+- The image has fewer requirements (e.g. ssh doesn't need to be installed).
+
+Disadvantages:
+
+- Not all Linux tools place nicely when run under chroot. (Though most of the most
+  common tools do play nicely since they are used to build Linux images under chroot.)
+  So, some customizations aren't possible to do using MIC.
+
+## Getting started
+
+1. Download a Mariner core image.
```

Comment:
Consider adding a link and possibly expanding to "... or use any existing Mariner image."

Reply:
There aren't any public links to the Mariner image file (e.g. vhd, vhdx) downloads.

Reply:
Ok, should we still mention that any already customized image can be used as an input as well?